src/ansys/fluent/core/file_session.py

```diff
@@ -810,7 +810,7 @@ def __init__(self):
         """__init__ method of FileSession class."""
         self._case_file = None
         self._data_file = None
-        self.monitors_manager = lambda: None
+        self.monitors = lambda: None
```

Comment:
This would still be accessed like a method call so still different from live sessions.